Sources/NetworkProtection/Diagnostics/NetworkProtectionConnectionBandwidthAnalyzer.swift

```diff
@@ -111,8 +111,16 @@ final class NetworkProtectionConnectionBandwidthAnalyzer {
 
     private func bytesPerSecond(newer: Snapshot, older: Snapshot) -> (rx: Double, tx: Double) {
         let deltaSeconds = newer.date.timeIntervalSince1970 - older.date.timeIntervalSince1970
```

Comment:
could this be replaced with `newer.date.timeIntervalSince(older.date)` for better readability?